content/concepts/introduction/overview.md

```diff
@@ -1,180 +1,126 @@
+from their "clients," as in the predominant
+[client-server model](https://en.wikipedia.org/wiki/Client%E2%80%93server_model).
+Instead, all peers in a P2P network are treated equally and can send and receive data
+to and from other peers.
+
+P2P networks can take many forms, including file-sharing systems like
+[BitTorrent](https://www.bittorrent.com/), blockchain networks like [Bitcoin](https://bitcoin.org/en/)
+and [Ethereum](https://ethereum.org/en/), and decentralized communication standards like
+[Matrix](https://matrix.org/). These systems all have different challenges and tradeoffs,
+but they share the goal of improving upon the traditional client-server networking model.
+
+libp2p was initially developed as part of the [InterPlanetary File System (IPFS)](https://ipfs.tech/)
+project as its wire protocol but has since phased out into a networking stack that has been adopted
+by a wide range of other projects as a networking layer. It provides a set of specifications that
+can be adapted to support various protocols, allowing libp2p applications to operate in diverse
+runtime and networking environments.
+
+In the context of the internet, the [OSI model](https://en.wikipedia.org/wiki/OSI_model) and the
+[TCP/IP model](https://en.wikipedia.org/wiki/Internet_protocol_suite) are conceptual models that provide
+a framework for understanding how networking protocols operate and interact. While these models offer
+a useful way to understand the functions and roles of different networking protocols, the actual
+implementations of networking protocols on the internet are often more complex and do not follow these
+models exactly.
+
+{{< alert icon="" context="info">}}
+If we consider the TCP/IP model, for instance, the primary focus of libp2p is on the
+Transport layer and the Application layer. However, these conceptual models have shortcomings.
+For example, certain tasks may be unnecessarily repeated across multiple layers, leading to
+inefficiencies. Additionally, some information may be hidden between layers, which can hinder
+opportunities for improvement.
+{{< /alert >}}
+
+## Why libp2p?
+
+libp2p was designed to address the limitations of traditional P2P networking approaches and existing
+network models, with the goal of enabling the distributed web. There are several reasons to
+consider using libp2p as a networking layer to create a robust P2P application:
+
+- **Modularlity**: libp2p is designed to be modular, allowing developers to mix and match different components
+  to meet the needs of their particular application. This makes it easy to customize the networking stack
+  to fit the specific requirements of any P2P application. libp2p also supports
+  [stream multiplexing](../multiplex/overview.md), which allows multiple streams of data to be transmitted
+  over a single connection, improving efficiency and enabling more advanced communication patterns.
```

Comment:
```suggestion
  to fit the specific requirements of any P2P application.
```
Can we remove this sentence? I don't know why stream multiplexing is called out specifically in the section about modularity